netctl/commands.go

```diff
@@ -55,6 +55,10 @@ var Commands = []cli.Command{
 						Name:  "policy, p",
 						Usage: "Policy List (separated by commas)",
 					},
+					cli.StringFlag{
+						Name:  "external-contracts, e",
+						Usage: "External contracts(separated by commas)",
```

Comment:
Allowing user to specify `-e` option multiple times will be more friendly. Please see https://github.com/contiv/netplugin/pull/386/files#diff-25b2dd1982b5db67097b2e5ee9891d9aR443 for an example of how to specify `cli.StringSliceFlag()` option.